data_steward/tools/delete_stale_test_datasets.py

```diff
@@ -97,8 +100,13 @@ def main(first_n):
     datasets_to_delete = _filter_stale_datasets(bq_client, first_n)
 
     for stale_dataset in datasets_to_delete:
+
         LOGGER.info(f"Running - bq_client.delete_dataset({stale_dataset})")
-        bq_client.delete_dataset(stale_dataset)
+
+        try:
+            bq_client.delete_dataset(stale_dataset)
+        except exceptions.BadRequest as e:
+            LOGGER.warning(f"Failed to delete {stale_dataset}. Message: {e}")
```

Comment:
I think `e` may not print the message. Maybe `e.message`?
```suggestion
            LOGGER.warning(f"Failed to delete {stale_dataset}. Message: {e.message}")
```